renderer/execute/wf_module.py

```diff
@@ -137,10 +137,17 @@ def _wrap_render_errors(render_call):
     try:
         return render_call()
     except ModuleError as err:
-        return RenderResult.from_deprecated_error(
-            "Something unexpected happened. We have been notified and are "
-            "working to fix it. If this persists, contact us. Error code: "
-            + format_for_user_debugging(err)
+        return RenderResult(
+            errors=[
+                RenderError(
+                    I18nMessage.trans(
+                        "py.renderer.execute.wf_module.wrap_render_errors",
+                        default="Something unexpected happened. We have been notified and are "
+                        "working to fix it. If this persists, contact us. Error code: {error_code}",
```

Comment:
Let's call this `{message}` (`{error_code}` sounds like it's an ID of some sort. We just say "error code" in our English to clarify to users that this isn't their problem.)